cfme/tests/ssui/test_ssui_myservice.py

```diff
@@ -128,7 +128,7 @@ def _finalize():
 
 @pytest.mark.long_running
 @pytest.mark.parametrize('context', [ViaSSUI])
-@pytest.mark.parametrize('order_service', [['console_test']], indirect=True)
+@pytest.mark.parametrize('order_service', [{'console_test': True}], indirect=True)
```

Comment:
Looks like this was the only other use of it, good.

Reply:
@nachandr @mshriver do we want to test Console Tests in the PRT as well for this change?

Reply:
PRT directive has been updated, thanks @nachandr